charts/kof-storage/values.yaml

```diff
@@ -5,6 +5,7 @@ global:
   random_password_length: 12
 
   # https://kubernetes.io/docs/concepts/storage/storage-classes/#default-storageclass
+  # PR Test Workflow
```

Comment:
Please remove any changes of this file, I assume it was used for testing, but you don't need to include these dummy changes into this PR